src/scraper.rs

```diff
@@ -105,7 +105,8 @@ impl Scraper {
             .into_iter()
             .filter(|candidate| Scraper::should_visit(candidate, &url))
             .for_each(|next_url| {
-                let next_full_url = url.join(&next_url).unwrap();
+                let mut next_full_url = url.join(&next_url).unwrap();
+                next_full_url.set_fragment(None);
```

Comment:
I would like to keep the fragment on the path (to get the full navigation experience after the clone) but remove it for the download. It should be as simple as moving line 109 after line 110

Reply:
@Skallwar actually, hold off on merging this, realizing moving those around doesn't work quite yet